### ISSUE TITLE
A bunch of suggestions to make the mod really impressive

### ISSUE THREAD

**Issue by santacapra** [wVibzz/ImmersiveWinds#31]
Ok here's a _bunch_ of ideas for you
Anyways, i don't expect you to do all of these, but i'm basically an idea generator so i figured i'd share lol. Take your pick :)

- Wind direction and speed not only affects smoke, but affects fire spread direction
- Wind direction variation, wind speed variation, and gusts (occasional higher speed. Irl its when multiple drafts converge i think but in game it doesnt have to be an actual simulation, could be randomized or something lol)
- Rarely, very strong winds, which can affect movement speed of land mobs
- Hurricanes? Very strong wind speed, which may affect mob speed, water may push you slightly/alot towards wind direction (more pushing the closer you are to the surface), rustling of leaves, grass, and tall grass... which reminds me...
- Sound effects. Leaves rustling, waves, etc, of varying volume and intensity would be super cool :D
- I saw someone else suggest wind directions. I agree with them, not all shaders have this, and I figured if its possible, a mod like this should override shaders in that way anyways
- Flying mobs (like bees) land during strong winds to wait it out (could also extend this to them finding shelter during Rain if you want. I know from irl experience they dont move even when hit by rain btw, they just stay perfectly still. Some info for you)
- Wind chimes... this might be hard to code. But it would be really cool to hear them chime against eachother and have outside your base. Like super cool!! It's stuff like this (and sound effects in general) which imo would make this mod make Minecraft feel so much more alive. And i feel like there is a huge lack of mods that make Minecraft feel alive.
- Get rid of the AI art... A screenshot is honestly so much better lol

Anyways I could come up with way more ideas but I'll hold off for now. Don't want to overwhelm you. Enjoy :D

**Comment by wVibzz** [comment]
1. Great idea will be using this
2. already a thing
3. played with this idea but it just dosent feel right, maybe something can be done with it not sure
4. far in the future but planned
5. future update
6. WIP
7. Cool idea ill play with it.
8. not sure if ill do this. maybe just add compatibility with a mod that already has one.
9. Plan on doing this after I fix the wind particles.

thanks for the feedback!

**Comment by santacapra** [comment]
Awesome!! Thanks and np

**Comment by wVibzz** [comment]
@santacapra also don't be afraid to suggest anything else I'll take everything into consideration. Plus im looking for more things to add before I push this next update out.

**Comment by santacapra** [comment]
For me, I've always wanted minecraft to feel more alive. If it had weather, seasons, realistic mob AI (like Rain World), and an immersive world (the world itself being alive; birth, death, growing and shrinking), and the weather and seasons was synced to a nice shader, it would completely change how the game feels.

Imo wind is only a small piece, but a very important one. I can't really say I have any suggestions other than work on making the next release as stable as possible ^^ Unless you're into dreaming big like me lol

**Comment by wVibzz** [comment]
0.8 update is pretty major so alot of these ideas of not been implemented for various reasons, although some of them will be planned for the 0.9 update. 

0.8 is releasing soon aswell!

**Comment by exocyt0sis** [comment]
> For me, I've always wanted minecraft to feel more alive. If it had weather, seasons, realistic mob AI (like Rain World), and an immersive world (the world itself being alive; birth, death, growing and shrinking), and the weather and seasons was synced to a nice shader, it would completely change how the game feels.

I fully agree with you - which is exactly why I always have Dynamic Trees and Regrowth installed along with Serene Seasons. They really make your world come alive for real, so please install them. Regarding weather, the recently released weather mod [Simple Clouds](https://modrinth.com/mod/simple-clouds) makes the competition pale - you'll get dynamic, volumetric clouds that sync across multiplayer servers meaning they are actual, physical objects! ❤️ Really hoping that Immersive Winds will be fully compatible with Simple Clouds in the near future by the way! 👍

**Comment by wVibzz** [comment]
I will look into compatibility! Just released 0.8 however so go check it out!
